skrf/media/media.py

```diff
@@ -1114,7 +1115,8 @@ def shunt_capacitor(self, C: NumberLike, **kwargs) -> Network:
         shunt_delay_short
         shunt_inductor
         """
-        return self.shunt(self.capacitor(C=C, **kwargs) ** self.short())
+        return self.shunt(self.capacitor(C=C, **kwargs) ** self.short(),
```

Comment:
I think `short` call also needs `**kwargs` or at least `s_def` to avoid warning of connecting different `s_def` networks when giving `s_def`.

Reply:
> I think `short` call also needs `**kwargs` or at least `s_def` to avoid warning of connecting different `s_def` networks when giving `s_def`.

Indeed. I will do that.

Reply:
Done